jsdrupal/drupal-admin-ui
# Issue #532 - Adjust Umami Demo patch to scrub remaining config dependencies from the Content Moderation / Workflow modules.

First Drupal contribution! 🙀 

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/532

## Screenshot / UI changes

None

## Testing instructions

The `remove-umami-workflows-dependency` patch now applies during `composer install` and the unmet config dependency issue is solved when running `composer setup`. The site now installs successfully and `composer start` brings it up locally as expected.
